frontend/js/behaviors/layeredImageViewer/layeredImageViewer.js

```diff
@@ -280,6 +336,29 @@ import OpenSeadragon from '../../libs/openseadragon4.min';
       .replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase());
   }
 
+  /**
+   * Transform an array of strings into a sentenace-like string
```

Comment:
Very minor things, but should this say sentence-like?

Reply:
Good spot, thanks for this I do like to be correct!